test/cluster/mzcompose.py

```diff
@@ -297,10 +297,11 @@ def workflow_test_builtin_migration(c: Composition) -> None:
             c.kill("materialized")
 
         with c.override(
-            # If this ever stops working, add the following argument:
-            # image="materialize/materialized:devel-438ea318093b3a15a924fbdae70e0db6d379a921"
-            # That commit added the missing column rolconnlimit to pg_authid.
-            Materialized(options=mz_options)
+            # This commit added the missing column rolconnlimit to pg_authid.
+            Materialized(
+                image="materialize/materialized:devel-438ea318093b3a15a924fbdae70e0db6d379a921",
```

Comment:
I'm real sad if this needs to happen, because it effectively means this test will never test any code in a PR, but just run exactly the same code over and over.

This test has caught migration bugs from other PRs before, and seems kinda good to have maybe? At least, I'd argue that if we want to hard code this we should maybe just...remove the test because it's doing nothing and replace it with a real migration test.

Reply:
I deleted it, it had a good run but with a breaking change it will no longer work. I'll open up an issue tonight about making adding a new upgrade test for this feature.